docs/application/web/guides/multimedia/media-controller.md

```diff
@@ -277,6 +284,60 @@ To manage the media controller features in your application, you must learn to s
 
       The `watcherId` variable stores the value, which can be used in the future to remove the listener from the server using the `removeCommandListener()` method.
 
+## Sending search request from the client to the server
+
+Client application can send search requests which consist of a collection of search conditions.
+Server can listen for incoming search requests and handle them accordingly.
+After handling a request, server can send a response to the client who sent that request.
+
+### Receiving Search Request
+
+To receive and handle search request on the server, follow these steps:
+
+```
+// 1. Get server handle.
+var server = tizen.mediacontroller.createServer();
+
+// 2. Define search request callback:
+function searchRequestCallback(client, request) {
+    console.log(client);
+    console.log(request);
+
+    // You can return RequestReply object, which will be sent to the client
+    // who can handle it in search request reply callback.
+    return new tizen.mediacontroller.RequestReply(123, {"someData": "someValue"});
+}
+
+// 3. Register search request listener
+server.setSearchRequestListener(searchRequestCallback);
+```
+
+### Sending Search Request
+
+To send the search request from the client application, follow these steps:
+
+```
```

Comment:
This is not a comment tag. This is a code section. I think it is easiest to understand this way for a developer who's trying to learn how to use this API.

Reply:
@m-michalski2, I understand your point, however, if we see the other sections, it has been given as a step. It looks readable and easy to understand. 

@mijin85cho, please let us know your views on this.

Reply:
I agree with @surbhig12's suggestion as for now this section only is using this listed code comment form.
@m-michalski2, would you please consider modifying it again?

Reply:
done, please verify. I have to note that i think the version where all steps were written as a single code example with comments is much better for this guide's user who can just copy/paste an entire example at once instead of having to copy each step one by one.